shelley/chain-and-ledger/formal-spec/incentives.tex

```diff
@@ -238,7 +242,7 @@ \subsection{General Parameters}
   \hline \hline
 \textbf{Parameter} & \textbf{Value} & \textbf{Description} & \textbf{Calculated as} \\\hline
 \textbf{\color{blue} $\textit{Ada}^{Tot}$ } & \textbf{\color{blue}  \ADA{45bn} } & \textbf{\color{blue}  The total \ada{} that could ever be created } & \textbf{\color{blue}  } \\\hline
-\textbf{\color{blue} \emph{P} } & \textbf{\color{blue}  20\%-50\% } & \textbf{\color{blue}  Participation Rate in the Shelley Incentivised TestNet } & \textbf{\color{blue}  } \\\hline
+%\textbf{\color{blue} \emph{P} } & \textbf{\color{blue}  100\% } & \textbf{\color{blue}  Participation Rate in the Shelley Incentivised TestNet } & \textbf{\color{blue}  } \\\hline
```

Comment:
better remove instead of commenting-out. Why is the number changed?

Reply:
I've left this commented in because it is still under discussion and may need to be reverted.  The number changed because this is what was anticipated by the researchers (in fact, it looks as if I will have to revert this).